examples/miscellaneous/plot_metadata_routing.py

```diff
@@ -373,56 +397,64 @@ def predict(self, X, **predict_params):
 
 # %%
 # - No metadata requested
-est = RouterConsumerClassifier(estimator=ExampleClassifier())
-print_routing(est)
+meta_est = RouterConsumerClassifier(estimator=ExampleClassifier())
+print_routing(meta_est)
 
 
 # %%
 # - ``sample_weight`` requested by underlying estimator
-est = RouterConsumerClassifier(
+meta_est = RouterConsumerClassifier(
     estimator=ExampleClassifier().set_fit_request(sample_weight=True)
 )
-print_routing(est)
+print_routing(meta_est)
 
 # %%
 # - ``sample_weight`` requested by meta-estimator
-est = RouterConsumerClassifier(estimator=ExampleClassifier()).set_fit_request(
+meta_est = RouterConsumerClassifier(estimator=ExampleClassifier()).set_fit_request(
     sample_weight=True
 )
-print_routing(est)
+print_routing(meta_est)
 
 # %%
 # Note the difference in the requested metadata representations above.
 #
 # - We can also alias the metadata to pass different values to them:
 
-est = RouterConsumerClassifier(
+meta_est = RouterConsumerClassifier(
     estimator=ExampleClassifier().set_fit_request(sample_weight="clf_sample_weight"),
 ).set_fit_request(sample_weight="meta_clf_sample_weight")
-print_routing(est)
+print_routing(meta_est)
 
 # %%
 # However, ``fit`` of the meta-estimator only needs the alias for the
 # sub-estimator, since it doesn't validate and route its own required metadata:
-est.fit(X, y, sample_weight=my_weights, clf_sample_weight=my_other_weights)
+meta_est.fit(
+    X, y, sample_weight=my_weights, clf_sample_weight=my_other_weights
+)  # <-- `meta_clf_sample_weight=my_weights`?
 
 # %%
-# - Alias only on the sub-estimator. This is useful if we don't want the
-#   meta-estimator to use the metadata, and we only want the metadata to be used
-#   by the sub-estimator.
-est = RouterConsumerClassifier(
+# - Alias only on the sub-estimator:
+#
+# This is useful when we don't want the meta-estimator to use the metadata, but
+# the sub-estimator should.
+meta_est = RouterConsumerClassifier(
     estimator=ExampleClassifier().set_fit_request(sample_weight="aliased_sample_weight")
-).set_fit_request(sample_weight=True)
-print_routing(est)
+).set_fit_request(
+    sample_weight=True
+)  # <-- why is `sample_weight=True` if `meta_est` shouldn't consume it?
+print_routing(meta_est)
```

Comment:
I don't understand the intend here.

 Why not simply set the sub-estimators `set_fit_requests` to `True` and the meta-estimator's to `False`? Why do we need an alias here at all? Why is `sample_weight=True` if the meta-estimator shouldn't consume it?